pt-PT/downloads.html

```diff
@@ -0,0 +1,81 @@
+---
+layout: page
+title: Bisq Downloads &lsaquo; Bisq - The decentralized Bitcoin exchange
+version: 1.1.2
+ref: donwloads
```

Comment:
2/2

Reply:
That's intended, it's to link the pages. The files inside pt-PT and their originals share a ref tag.

Reply:
Okay, so I'm 100% clear, it's not misspelled? If it's good then I can merge now.

Reply:
@m52go oh, it is misspelled.